source/digits_hits/src/GateTrajectoryNavigator.cc

```diff
@@ -183,17 +183,24 @@ std::vector<G4int> GateTrajectoryNavigator::FindAnnihilationGammasTrackID()
       // considered by GateAnalysis
       G4int i1;
       G4int i2;
+      G4int i3;
+      bool only2gamma = true;
+      
       for (G4int j1=0; j1<nPh; j1++) {
 	for (G4int j2=j1+1; j2<nPh; j2++) {
+    for(G4int j3=j2+1; j3<nPh; j3++){
```

Comment:
Make a new method for this 3 loops and correct the tabs.
Do you understand why you make this? Remember two things:
1. 3 gammas from the orthopositronium (2 gammas from parapositronium) annihilation have the same vertex
2. prompt gamma has a different vertex

See: https://github.com/OpenGATE/Gate/blob/develop/source/physics/src/GatePositroniumDecayModel.cc#L117

Reply:
Done